api/elastic.py

```diff
@@ -227,6 +232,28 @@ def titleQuery(self, titleText):
                 )
             )
         ])
+    
+    def identifierQuery(self, identifierText):
+        self.searchedFields.extend(['identifiers', 'editions.identifiers'])
+
+        return Q('bool', should=[
+            Q(
+                'query_string',
+                query=identifierText,
+                fields=['identifiers.*'],
```

Comment:
This will definitely work, but I wonder if we should have it work along the lines of the `authorityQuery` where we break out the identifier authority and value and look up that way. It's a bit more complicated but I think would be more accurate and precise.

Unless we already discussed and decided to take a different approach, I'm not sure

Reply:
It would mean splitting the string and handling those two values separately, but I don't think that's a crazy amount of complexity to add

Reply:
From my TAD, we decided to have one query for the user to search for a work just based on the identifier value and another query for the user to search based on the authority and value. Based on this, I believe the identifierQuery method can stay the same, but we can add an authIdentQuery method that does what you suggest with splitting up a string and handling the two values.

Reply:
I think that works! Another approach would be to have a method that checks for the presence of an authority and if present adds that as part of the query and if not just does the search by identifier value, but I leave that up to you!